documentation/modules/ROOT/pages/configuration/filtering.adoc

```diff
@@ -40,62 +47,108 @@ transforms.filter.condition=value.op == 'u' && value.before.id == 2
 ...
 ----
 
-In this example we are using `Groovy` as expression language, and we're filtering out all update records with `id` field equal to `2`.
+The preceding example specifies the use of the `Groovy` expression language.
+The regular expression `value.op == 'u' && value.before.id == 2` removes all messages, except those that represent update (`u`) records with `id` values that are equal to `2`.
 
 [IMPORTANT]
 ====
-{prodname} does not come with the language implementations in its installation packages.
-It is the user's responsibility to provide an implementation, such as link:https://groovy-lang.org/[Groovy 3] or link:https://github.com/graalvm/graaljs[GraalVM JavaScript], on the classpath.
-Bootstrapping is done exclusively via the JSR 223 API currently, so the engine's support for this API must be provided as well.
+Although the {prodname} package includes the JSR 223 API, which provides support for scripting languages that integrate with JSR 223, by default {prodname} cannot interpret scripts from these languages.
+To use an expression language, you must download the JSR-223 script engine JAR for the language, and add the JAR to your classpath, along any other JAR files used by the language implementation.
+For Groovy 3, you can download the JAR from https://groovy-lang.org/. For GraalVM JavaScript, the JAR is available at https://github.com/graalvm/graaljs.   
+
+Typically, you add the language JAR to the directory that contains the {prodname} connector JAR files, which is the directory that is referenced in the `plugin.path` configuration property that is set for the Kafka Connect plugin. 
+If you install the script engine JAR to a different directory, you must add that directory to the list in the `plugin.path` property. 
+
 ====
 
-{prodname} binds four variables into the evaluation context:
+// Type: concept
+// ModuleID: variables-for-use-in-filter-expressions
+== Variables for use in filter expressions
+
+When you create expressions to specify filter conditions, you can use the variables that {prodname} binds into the evaluation context.
+Certain variables are bound into the evaluation context for the SMT.
+By binding variables , {prodname} enables the SMT to look up and interpret their values as it evaluates the conditions in an expression. 
```

Comment:
```suggestion
By binding variables, {prodname} enables the SMT to look up and interpret their values as it evaluates the conditions in an expression. 
```

Reply:
I made this change as part of the latest commit.